docs/docs/integrating-with-other-libraries.md

```diff
@@ -0,0 +1,270 @@
+---
+id: integrating-with-other-libraries
+title: Integrating with other libraries
+permalink: docs/integrating-with-other-libraries.html
+---
+
+React can be used in any web application. It can be embedded in other applications and, with a little care, other applications can be embedded in React. This guide will examine some of the more common use cases, focusing on integration with jQuery and Backbone.
+
+## Integrating with DOM Manipulation Plugins
+
+React is unaware of changes made to the DOM outside of React. It determines updates based on its own internal representation, and if those updates are invalidated, React has no way to recover.
+
+This does not mean it is impossible or even necessarily difficult to combine React with other ways of affecting the DOM, you just have to be mindful of what each are doing.
+
+The easiest way to avoid conflicts is to prevent the React component from updating. This can be done explicitly by returning false from [`shouldComponentUpdate()`](/react/docs/react-component.html#shouldcomponentupdate), or by rendering elements that have no reason to change.
```

Comment:
And I see that you explained it later, but generally people stumble where they're confused, and don't read past it because they're not sure if they'll be able to follow the rest. It's better to explain before rather than after the code.